src/Joint.cc

```diff
@@ -354,6 +355,14 @@ std::optional<std::vector<double>> Joint::Position(
       this->dataPtr->id);
 }
 
+//////////////////////////////////////////////////
+std::optional<std::vector<gz::math::Vector2d>> Joint::VelocityLimits(
+    const EntityComponentManager &_ecm) const
+{
+  return _ecm.ComponentData<components::JointVelocityLimits>(
+      this->dataPtr->id);
```

Comment:
I think you can just get this information from the `JointAxis` component, e.g.

```cpp
auto jointAxis = _ecm.Component<components::JointAxis>(this->dataPtr->id);
if (jointAxis)
{
  return jointAxis->Data().MaxVelocity();
}
```

For joint velocity, there is no lower or upper limit, but only just one [velocity limit field](http://sdformat.org/spec?ver=1.12&elem=joint#limit_velocity)